griptape/tokenizers/anthropic_tokenizer.py

```diff
@@ -6,13 +6,16 @@
 @define(frozen=True)
 class AnthropicTokenizer(BaseTokenizer):
     DEFAULT_MODEL = "claude-2"
-    DEFAULT_MAX_TOKENS = 100000
-
+    MODEL_TO_MAX_TOKENS = {
+        "claude-2": 100000,
+        "anthropic.claude-v2": 8192
```

Comment:
This feels a bit awkward. Don't really know what to do about it.

Reply:
Would something like [this](https://github.com/griptape-ai/griptape/blob/main/griptape/tokenizers/tiktoken_tokenizer.py#L17) be any better?